src/test/java/org/broadinstitute/hellbender/tools/PrintBGZFBlockInformationIntegrationTest.java

```diff
@@ -100,4 +100,20 @@ public void testRegularGzipFile() throws IOException {
         };
         runCommandLine(args);
     }
+
+    /* Make sure that we can handle a standard BAM file */
```

Comment:
Maybe add a test to ensure there is a nice exception when the user provides a non-BGZF file (maybe a VCF).

Reply:
Done